### PR TITLE
Bug541873 - ENTITYMANAGER.DETACH() TRIGGERS LAZY LOADING INTO THE PER…

### PR DESCRIPTION
**Pull request by sureshbn**
…SISTENCE CONTEXT

Signed-off-by: Sureshkumar Balakrishnannair <sureshkumar.balakrishnannair@oracle.com>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=541873

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=541873

**Comment by lukasj** [comment]
```
[junit] Test org.eclipse.persistence.testing.tests.jpa.advanced.AdvancedServerTestSuite FAILED
```
can you take a look why? or try to rebase the change on master and force push to trigger new run, please

**Comment by sureshbn** [comment]
In local didn't face any issue  - Done a fresh checkout previously . As suggested let me try a re-base/force pull again

**Comment by sureshbn** [comment]
fyi..I initiated a new one

**Comment by sureshbn** [comment]
Thanks for the comment .  Integrated with Server and corrected the test-lrg-server

File: jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/tests/jpa/advanced/EntityManagerJUnitTestSuite.java [hunk 1]
```diff
@@ -520,15 +525,16 @@ public static Test suiteSpring() {
...
 
         Collections.sort(tests);
         for (String test : tests) {
             suite.addTest(new EntityManagerJUnitTestSuite(test));
         }
-        if (!isJPA10()) {
+        /*if (!isJPA10()) {
             suite.addTest(new EntityManagerJUnitTestSuite("testCascadeDetach"));
-        }
+        }*/
```
Comment (lukasj) [inline jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/tests/jpa/advanced/EntityManagerJUnitTestSuite.java]: why is this test commented out?

Comment (sureshbn) [reply jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/tests/jpa/advanced/EntityManagerJUnitTestSuite.java]: Its was not part of the change , done for some local change . Let me re-build

File: jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/models/jpa/advanced/Material.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```
Comment (lukasj) [inline jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/models/jpa/advanced/Material.java]: this is not valid header, it must contain EPL/EDL., see ie https://github.com/eclipse-ee4j/eclipselink/blob/master/project-admin/Sample.java for sample

Comment (sureshbn) [reply jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/models/jpa/advanced/Material.java]: Changed the headers , anything Pls let me know

**Pull request merged**